### DISCUSSION TITLE
Spatial ST_Transform: Convert between vertical CRS

### DISCUSSION THREAD

**Discussion by bertt** [duckdb/duckdb#14345] [Ideas]
Hi,

I noticed in DuckDB 1.1.1 that conversion between different vertical CRS does not work.

Sample query in PostGIS:

```
select ST_AsText(ST_Transform(ST_GeomFromText('POINT(121302 487371 2.68)', 7415), 4979));
result: POINT Z (4.892367035931109 52.37317920269912 45.66258579945144)
```

Sample query in DuckDB:

```
select st_astext(st_transform(ST_GeomFromText('POINT Z(121302 487371 2.68)'), 'epsg:7415','epsg:4979'));
result: POINT Z (52.37419207762104 4.892729187131278 2.68) 
```

Notice the vertical value (2.68) is converted to 45.66258579945144 in PostGIS, but not in the DuckDB case.

Idea: Support conversions between different vertical CRS in DuckDB

**Comment by Maxxen** [comment]
Hello! Thanks for opening this issue!
We currently don't take the z-axis into account when projecting, but I agree that's definitely something we should support!
